python/cudf/cudf/core/dataframe.py

```diff
@@ -7883,6 +7881,16 @@ def interleave_columns(self):
             )
         return self._constructor_sliced._from_column(result_col)
 
+    @acquire_spill_lock()
+    def _compute_columns(self, expr: str) -> ColumnBase:
```

Comment:
Suggesting because it returns one column
```suggestion
    def _compute_column(self, expr: str) -> ColumnBase:
```

Reply:
Sure, that makes sense. I'll incorporate this in another PR just to save a CI run

Reply:
I incorporated it in https://github.com/rapidsai/cudf/pull/17456